Fmglobal

oluwi opened the pull request:
Sign change for v-bar

rmcdermo commented:
Luwi,
Thanks for trying.  But you did not update your repository first and so you reintroduced errors I had fixed before.  So I cannot merge this.  It's ok, I'm almost done updating you stuff.

oluwi commented:
Randy, My apologies. I was sure I updated before modifying... something must have gone wrong. Thanks for helping to clean it up.
Luwi


⁣ ​

On Apr 20, 2021, 6:10 PM, at 6:10 PM, Randy McDermott ***@***.***> wrote:
>Luwi,
>Thanks for trying.  But you did not update your repository first and so
>you reintroduced errors I had fixed before.  So I cannot merge this. 
>It's ok, I'm almost done updating you stuff.
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/MaCFP/macfp-db/pull/257#issuecomment-823631902

rmcdermo commented:
No problem, Luwi.  Thanks for the contribution.
